src/Components/MainPage/TrendList/trendlist.css

```diff
@@ -0,0 +1,164 @@
+.trending-container {
+  max-width: 1250px;
+  width: 80vw;
+  margin: 0 auto;
+}
+
+.trending-title-wrapper {
+  display: flex;
+  text-transform: uppercase;
+  align-items: center;
+}
+
+.trending-title {
+  margin-right: 30px;
+  color: #493871;
+}
+
+.btn {
+  font-family: "Oswald", sans-serif;
+  background-color: transparent;
+  color: #493871;
+  border: 2px solid #493871;
+  padding: 9px;
+  letter-spacing: 5px;
+  border-radius: 10px;
+  outline: 0;
```

Comment:
Outline should be there for accessibility reasons. If you'd go over your page with a tab, then you wouldn't know if you are located on a button, as you don't have any outline there. You can style the focus atate differently, if you don't like the blue outline, but it's not a good practice to lose it all together :) 
For example for your searchbar component you have a separate focus style, so even though there isn't any outline, I can still see that this component is focused.